bindings/pyfiction/include/pyfiction/algorithms/verification/design_rule_violations.hpp

```diff
@@ -39,7 +40,8 @@ void gate_level_drvs(pybind11::module& m)
 
             return {stats.warnings, stats.drvs};
         },
-        "layout"_a, "params"_a = fiction::gate_level_drv_params{}, "print_report"_a = false);
+        "layout"_a, "params"_a = fiction::gate_level_drv_params{}, "print_report"_a = false,
+        DOC(fiction_detail_gate_level_drvs_impl));
```

Comment:
I might be mistaken, but isn't this the wrong docstring? Shouldn't it be something like `fiction_gate_level_drvs`?